sdk/src/driftClient.ts

```diff
@@ -2076,54 +2084,96 @@ export class DriftClient {
 	): Promise<{ txSig: TransactionSignature; signedFillTx: Transaction }> {
```

Comment:
i think some wallets might not work with versioned transactions 🤔 if so, we might want the ui to conditionally use versioned tx only if wallet is compatible